streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorStateManagerTest.java

```diff
@@ -658,6 +668,17 @@ public void flush() {
                 flushedStore.set(true);
             }
         };
+        final ProcessorStateManager stateManager = new ProcessorStateManager(
+            taskId,
+            Collections.singleton(changelogTopicPartition),
+            false,
+            stateDirectory,
+            singletonMap(storeName, changelogTopic),
+            asList(stateStore1.name(), stateStore2.name()),
+            changelogReader,
+            false,
+            logContext);
+
```

Comment:
I had to move a couple of these so that I could list the store names in the topological order (I.e., I have to generate the names before I can list them).